documentation/book/con-requests-kafka-bridge.adoc

```diff
@@ -0,0 +1,100 @@
+// Module included in the following assemblies:
+//
+// assembly-using-the-kafka-bridge.adoc
+
+[id='con-requests-kafka-bridge-{context}']
+= Requests to the {ProductName} Kafka Bridge
+
+== Authentication and encryption
+
+Authentication and encryption between external clients and the Kafka Bridge is not yet supported. This means that requests sent from external clients to the Kafka Bridge are:
+
+* Not encrypted, and must use HTTP rather than HTTPS
+
+* Sent without authentication
+
+Within your {ProductPlatformName} cluster, you can configure TLS or SASL-based authentication between the Kafka Bridge and your Kafka cluster. For more information, see LINK TO CHAPTER 3 **Connecting to Kafka brokers with Authentication**. 
+
+== Data formats and headers
+
+Use the following information to help you submit valid requests to the Kafka Bridge. 
+
+=== Content Type headers
+
+API request and response bodies are always encoded as JSON. 
+
+* When performing consumer operations, `POST` requests must provide the following `Content-Type` header:
++
+[source,http,subs=+quotes]
+----
+Content-Type: application/vnd.kafka.v2+json
+----
+
+* When performing producer operations, `POST` requests must provide the following `Content-Type` header specifying the *embedded data format* of the consumer, either `json` or `binary`:
```

Comment:
I would not say "of the consumer". The producer has the freedom to send the data in the embedded format which makes sense to the application. It's up to the consumer to adhere to such format in order to get the data and understanding its content.